jax/experimental/jax2tf/jax2tf.py

```diff
@@ -840,6 +841,8 @@ def _unexpected_primitive(p: core.Primitive, *args, **kwargs):
   lax_parallel.axis_index_p, lax_parallel.pdot_p,
 
   pxla.xla_pmap_p,
+
+  sharded_jit.sharding_constraint_p
```

Comment:
I think [this commit](https://github.com/google/jax/commit/a04f8197cae1434840f9badc5d931250bc80a153) broke my tests locally. I guess it does not happen in google3, but just to be sure… :)